### PR TITLE
WIP: Change color or table when changed

### LINKED ISSUE #41: Tabellenansicht - farbige Zeile
Sobald in der Tabellenansicht Werte in einer Zeile verändert wurden (direkt beim tippen, noch vor dem Speichern) sollte sich diese Zeile dezent einfärben um auf Änderungen aufmerksam zu machen. Beim speichern soll die Färbung wieder verschwinden.

- [ ] Einfärben bei Änderung
- [ ] Farbe beim speichern wieder entfernen

### PR DESCRIPTION
**Pull request by Helene0**
This closes #41 .

I'm stuck on this PR because of two things:
  - `md-select` doesn't trigger change events properly (and I don't know how to catch them), so changes on the category and license aren't registered.
  - `v-bind:class` does not change the form's class when the variable is changed. It needs two changes to become active, which defeats the purpose.

Can someone please looks at this and hopefully find an easy fix.

### CODE REVIEW

**Comment by adrianjost** [comment]
@Helene0 you could watch for changes in the data itself. 
You can find an implementation [here](https://github.com/schul-cloud/schulcloud-content-editor/blob/master/src/components/paginationTemplate.vue#L35)
In your case: watch for `data` (or `contentData` if your branch is up to date) and set `this.data.isDirty = true;`

File: src/components/contentTableRow.vue [hunk 1]
```diff
@@ -65,6 +70,8 @@ export default {
         alert('Correct the errors!');
     },
     submitContent: function (event) {
+        this.data.isDirty = false;
```
Comment (adrianjost) [inline src/components/contentTableRow.vue]: use `this.isDirty`

File: src/components/contentTableRow.vue [hunk 2]
```diff
@@ -74,7 +81,12 @@ export default {
                 headers: {"Authorization" : "Bearer " + localStorage.getItem('jwt')},
             });
         }
-    }
+    },
+    itemChanged() {
+      console.log('blaaa');
+      this.data.isDirty = true;
```
Comment (adrianjost) [inline src/components/contentTableRow.vue]: use `this.isDirty` here too

File: src/components/contentTableRow.vue [hunk 3]
```diff
@@ -1,5 +1,5 @@
 <template>
-  <form id="contentForm" @submit.prevent="validateBeforeSubmit">
+  <form id="contentForm" @submit.prevent="validateBeforeSubmit" @change="itemChanged" v-bind:class="{ changed: data.isDirty }">
```
Comment (adrianjost) [inline src/components/contentTableRow.vue]: `changed: isDirty` should work better

**Comment by Helene0** [comment]
@adrianjost thanks for the hints. this should work now

**Pull request merged**